CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+# Welcome to contributing guide <!-- omit in toc -->
+
+Thank you for investing your time in contributing to our project!
+
+Read our
+[Code of Conduct](https://www.contributor-covenant.org/version/2/0/code_of_conduct/)
+to keep our community approachable and respectable.
+
+In this guide you will
+get an overview of the contribution
+workflow from opening an issue, creating a PR, reviewing, and merging the PR.
+
+Use the table of contents icon on the top left corner
+of this document to get to a specific section of this guide quickly.
+
+## New contributor guide
+
+To get an overview of the project,
+read the [README](README.md). Here are some resources
+to help you get started with open source contributions:
+
+- [Finding ways to contribute to open source on GitHub](https://docs.github.com/en/get-started/exploring-projects-on-github/finding-ways-to-contribute-to-open-source-on-github)
+- [Set up Git](https://docs.github.com/en/get-started/quickstart/set-up-git)
+- [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow)
+- [Collaborating with pull requests](https://docs.github.com/en/github/collaborating-with-pull-requests)
+
+### Issues
+
+#### Create a new issue
+
+If you spot a problem with this template,
+[search if an issue already exists](https://docs.github.com/en/github/searching-for-information-on-github/searching-on-github/searching-issues-and-pull-requests#search-by-the-title-body-or-comments).
+If a related issue doesn't exist, you can open a new issue using a relevant [issue form](https://github.com/VilnaCRM-Org/frontend-ssr-template/issues/new).
```

Comment:
We need to change links from frontend-ssr-template to the current GitHub repository (frontend-spa-template)